src/styles/blocks/head-block/nav.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+  }
+
+  &__item {
+    list-style: none;
+    display: flex;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__item a {
+    display: inline-block;
+    text-decoration: none;
+    font-size: 12px;
+    font-weight: 500;
+    height: 60px;
+    line-height: 60px;
+    color: $nav_text-color;
+    cursor: pointer;
+    text-transform: uppercase;
+
+    &.is-active {
+      color: $main-blue-color;
+    }
+
+    &.is-active::after {
```

Comment:
The 'is-active::after' pseudo-element can be affected by the layout changes. Consider using a solid border-bottom on the 'is-active' class instead.